utils/scripts/helpers/scan_daos_maldet.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -uex
+
+# Install the DAOS RPMs and then do a malware scan
+# of the resulting system volume.
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# shellcheck source=utils/scripts/helpers/distro_info.sh
+source "$mydir/distro_info.sh"
+
+if command -v dnf; then
+  sudo dnf install \
+    daos{,-{client,server,tests,debuginfo,devel}}-"${DAOS_PKG_VERSION}"
+elif command -v apt-get; then
+  echo "Ubuntu not implemented yet."
+  exit 1
+else
+  echo "Unknown distribution."
+  exit 1
+fi
+fails=0
+errs=0
+mal_strt="!-- "
+mal_end=" --"
+# debug
+ls -l /usr/bin/md5sum | true
```

Comment:
(lint) Use find instead of ls to better handle non-alphanumeric filenames. [SC2012]
(lint) Piping to 'true', a command that doesn't read stdin. Wrong command or missing xargs? [SC2216]